Formula/purescript-language-server.rb

```diff
@@ -0,0 +1,21 @@
+require "language/node"
+
+class PurescriptLanguageServer < Formula
+  desc "Node-based Language Server Protocol server for PureScript"
```

Comment:
Should this be made shorter?

```suggestion
  desc "Node-based LSP server for PureScript"
```

Maybe even "Node-based" could be removed?

Reply:
yeah, `node-based` seems redundant, but `Language Server Protocol` is certainly better than `LSP`

Reply:
Done.